roofit/roofitcore/src/RooProjectedPdf.cxx

```diff
@@ -95,7 +95,7 @@ double RooProjectedPdf::evaluate() const
 {
   // Calculate current unnormalized value of object
   int code ;
-  return getProjection(&intobs, _normSet, 0, code)->getVal() ;
+  return getProjection(&intobs, _normSet, (_normRangeOverride.Length()>0 ? _normRangeOverride.Data() : (_normRange.Length()>0 ? _normRange.Data() : 0)), code)->getVal() ;
```

Comment:
```suggestion
  return getProjection(&intobs, _normSet, normRange(), code)->getVal() ;
```

Pretty good improvement! I'm always happy to see RooFit handling the normalization ranges more consistently.

My only doubt is about this `_normRangeOverride`. Right now it's only used in one place (`RooAbsPdf::syncNormalization()`), which is also inconsistent because for other integrals it's not considered, also if they are used as normalization integrals.

I was thinking about getting rid of `_normRangeOverride`, and this PR goes in the other direction, unfortunately.

What do you think about this?

Reply:
this is fine with me - I only put normRangeOverride in because I thought I had to support it and I didnt check how extensively it was used, or not as you claim. so happy to take your suggestion instead!